src/quo/components/tags/summary_tag/component_spec.cljs

```diff
@@ -0,0 +1,51 @@
+(ns quo.components.tags.summary-tag.component-spec
+  (:require
+    [quo.components.tags.summary-tag.view :as summary-tag]
+    [test-helpers.component :as h]))
+
+(h/describe "Summary tag component tests"
```

Comment:
I'd suggest adding one more test where the component instantiated with empty props, to make sure it doesn't crash

Reply:
Similarly, since we have a set of props packed under `:token-props`, `:account-props`, their empty state also worth testing.
Right now component crashes the app in following scenarios:
```clojure
[quo/summary-tag
         {:type        :token
          :token-props {}}]
```
this also:
```clojure
[quo/summary-tag
         {:type        :account
          :account-props {}}]
```          
          
Didn't test the rest